include/zer_api.h

```diff
@@ -86,6 +86,12 @@ extern "C" {
 /// @brief compiler-independent type
 typedef uint8_t zer_bool_t;
 
+///////////////////////////////////////////////////////////////////////////////
+/// @brief callback function with user data
+typedef void (zer_func_callback_t)(
```

Comment:
Should we update `zerModuleCreate` to take this type instead of the `void **` it currently accepts?

Reply:
Done